docs/nidcpower/enums.rst

```diff
@@ -576,7 +576,7 @@ Enums used in NI-DCPower
 
 
 
-    .. py:attribute:: nidcpower.TriggerType.DIGITAL
+    .. py:attribute:: nidcpower.TriggerType.DIGITAL_EDGE
```

Comment:
Is this right?

Before:

TriggerType
* DIGITAL
* SOFTWARE

Now

TriggerType
* DIGITAL_EDGE
* SOFTWARE_EDGE

Reply:
Yes, because _EDGE is not common with all names. There is also a `NONE` name.

Reply:
So the code is doing what it is supposed to do. But do we want _EDGE in these cases? In other places I remember seeing just as DIGITAL and SOFTWARE.

Reply:
Once this is in, we can go through all the enums and see if there are any names we want to explicitly override. There may be others in addition to this one. At that point we will have to decide if the better name is worth hand maintaining it.

Reply:
ok